tests/integration/helpers.py

```diff
@@ -399,6 +450,32 @@ async def get_application_hosts(ops_test: OpsTest, app_name: str, units: List[st
     return hosts
 
 
+async def check_full_status(
+    ops_test: OpsTest, app: str = APP_NAME, status: str = "active", status_msg: str | None = None
+) -> bool:
+    """Compare app and unit status against those requested in the parameters."""
+    status_data = await ops_test.model.get_status()  # noqa: F821
+
+    if not status_data.applications[app].status.status == status:
```

Comment:
Can you confirm that `.status.status` is correct here?

Reply:
![image](https://github.com/user-attachments/assets/149e6497-4222-43d9-8816-20eec43e160b)

Yes, it is correct.